sql/updates/world/master/2024_01_28_03_world_priest_protective_light.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM `spell_proc` WHERE `SpellId` IN (193063);
+INSERT INTO `spell_proc` (`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`ProcFlags2`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`DisableEffectsMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`) VALUES
+(193063,0x02,6,0x00000800,0x00000000,0x00000000,0x00000000,0x0,0x0,0x2,0x2,0x3,0x0,0x0,0,0,0,0); -- Protective Light
+
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (193063);
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (193063, 'spell_pri_protective_light');
```

Comment:
we put the values on the next line, makes it easier to read
```suggestion
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
(193063, 'spell_pri_protective_light');
```